tests/test__helpers.py

```diff
@@ -49,17 +50,35 @@ def test_list_input(self):
     self.assertEqual(result, """[42,1337]""")
 
 
+class Test__to_bytes(unittest.TestCase):
+
+  def test_with_bytes(self):
+    value = b'bytes-val'
+    self.assertEqual(_to_bytes(value), value)
+
+  def test_with_unicode(self):
+    value = u'string-val'
+    encoded_value = b'string-val'
+    self.assertEqual(_to_bytes(value), encoded_value)
+
+  def test_with_nonstring_type(self):
+    value = object()
+    self.assertRaises(ValueError, _to_bytes, value)
+
+
 class Test__urlsafe_b64encode(unittest.TestCase):
 
+  DEADBEEF_ENCODED = b'ZGVhZGJlZWY'
```

Comment:
Module-scope constant rather than class-scope constant? Any reason for this to be at class scope?